modules/API/external/external_api.js

```diff
@@ -81,6 +81,26 @@ function changeParticipantNumber(APIInstance, number) {
 }
 
 /**
+ * Generates array with URL params based on the passed config object that will
+ * be used for the Jitsi Meet URL generation.
+ *
+ * @param config {object} the config object.
+ * @returns {Array<string>} the array with URL param strings.
+ */
+function configToURLParamsArray(config) {
+    const params = [];
+    Object.keys(config).forEach( key => {
```

Comment:
Agree, but what I don't like is that theoretically with the for..in we will get the prototype properties and with `Object.key` we won't, right? This way we don't need the `if` for the string. That's why I prefer the code with `Object.key`.